tests/codeception/functional/NumberingButtonCest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace wiki\functional;
+
+use humhub\modules\space\models\Space;
+use humhub\modules\wiki\helpers\Url;
+use humhub\modules\wiki\models\WikiPage;
+use humhub\modules\wiki\Module;
+use humhub\modules\user\models\User;
+use wiki\FunctionalTester;
+use Yii;
+
+class NumberingButtonCest
+{
+    public function testNumberingButtonHtml(FunctionalTester $I)
+    {
+        $I->wantTo('check if the numbering button toggles between enabled and disabled states at page view');
+        $space = $I->loginBySpaceUserGroup(Space::USERGROUP_MEMBER);
+
+        $I->amAdmin(true);
+        $I->enableModule($space->guid, 'wiki');
+        $category = $I->createWiki($space, 'Test Wiki Page', 'Test Wiki Page content');
+
+        $I->amOnSpace($space->guid, '/wiki/page/view', ['id' => $category->id, 'title' => $category->title]);
+
+        $module = Yii::$app->getModule('wiki');                      
+        $user = Yii::$app->user->identity;
+        $numberingEnabled = $module->settings->contentContainer($user)->get('wikiNumberingEnabled');
+
+        $I->seeElement('a.toggle-numbering');
+        if ($numberingEnabled) {
+            $I->see('Disable Numbering', 'a.toggle-numbering');
+        }   
+        else {
+            $I->see('Enable Numbering', 'a.toggle-numbering');
+        } 
```

Comment:
The test already looks very good. Well done.
However, IF statements should be avoided in unit tests. It would be better to ensure a fixed condition at the beginning of the test (e.g. numbering switched off), then trigger an action and check for a fixed expected value.

However, you can execute several actions in succession in one test, e.g.
1. start condition: Numbering switched off
2. action: press button
3. check whether numbering is now switched on
4. action: press button
5. check whether numbering is now switched off